### PR TITLE
Glossary #1

### PR DESCRIPTION
**Pull request by justinpoh**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -795,6 +795,64 @@ Private contact detail
 A contact detail that is not meant to be shared with others
 ....
 
+
+[[contact-information]]
+Contact information of a person
```
Comment (chuaweiwen) [inline docs/DeveloperGuide.adoc:800]: Capitalise the title "Contact Information of a Person"

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -795,6 +795,64 @@ Private contact detail
...
+A computer programming principle that states that every module or class should have responsibility over only a single part of functionality of the program.
+....
+
+[[OCP]]
+Open-Close Principle
+
+....
+An object-oriented programming principle that states that software entities such as classes and modules should be open for extension, but closed to modification.
+....
+
```
Comment (tshradheya) [inline docs/DeveloperGuide.adoc:862]: Add Frequently Visited Contacts

Comment (tshradheya) [inline docs/DeveloperGuide.adoc:862]: Add `Data File`

- Stores data of contacts in XML format

**Review by tshradheya — COMMENTED** [review]
Check if we can add more terms which are used in the App

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (justinpoh) [reply docs/DeveloperGuide.adoc:800]: I was following the earlier style so I don't really know if it should be capital or not. However, in the HTML appearence, the text displayed is the second line, not the text within the bracket.

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (justinpoh) [reply docs/DeveloperGuide.adoc:862]: Thats a good one. Will do that

Comment (justinpoh) [reply docs/DeveloperGuide.adoc:862]: We have not agreed on a convention yet though. "Data File" or "Source File"

Comment (tshradheya) [reply docs/DeveloperGuide.adoc:862]: `Data File` will make more sense as we plan to implement share function

**Pull request merged**